apollo-client/src/main/java/com/ctrip/framework/apollo/spi/DefaultConfigFactory.java

```diff
@@ -111,7 +101,11 @@ public ConfigFile createConfigFile(String namespace, ConfigFileFormat configFile
   }
 
   ConfigRepository createConfigRepository(String namespace) {
-    if (m_configUtil.isPropertyFileCacheEnabled()) {
+    // TODO 本地和configMap允许同时开启
+    // 已完成，创建configmapRepo时会同时创建一个fileRepo作为上游，相当于同时开启了，路径若未设置用默认
+    if (m_configUtil.isPropertyKubernetesCacheEnabled()) {
+      return createConfigMapConfigRepository(namespace);
+    }else if (m_configUtil.isPropertyFileCacheEnabled()) {
```

Comment:
Please address the formatting issues in accordance with the [Contributing Guide](https://github.com/apolloconfig/apollo/blob/master/CONTRIBUTING.md).
```suggestion
    } else if (m_configUtil.isPropertyFileCacheEnabled()) {
```